spec/factories/course_groups.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :course_group, class: Course::Group.name do
+    course
+    creator
+    updater
+    name "group 1"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.